acoustic.ipynb

```diff
@@ -313,7 +321,7 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "remarks = f.match('^R[0-9]{1,2}', type='parameter')\n",
+    "remarks = [par for par in f.parameters if re.match('^R[0-9]{1,2}', par.name)]\n",
```

Comment:
You can use `f.find` here. It works identically to the old `match`. The only difference is the argument order. In `find` type is the 1st argument, and name is the 2nd. I.e. opposite of the old `match`

Reply:
Neat! It's best to use `f.find` here, so that we have examples of the similar functions `f.find` and `f.object` in the notebook.